pupil_src/shared_modules/calibration_routines/calibration_plugin_base.py

```diff
@@ -18,7 +19,9 @@ def on_notify(self,notification):
 
         Emits notifications:
             ``calibration.started``: Calibration procedure started
-            ``calibration.stopped``: Calibration procedure stopped
+            ``calibration.stopped``: Calibration procedure stopped (manually)
```

Comment:
may also be stopped automatically. So the comment is to much and potentially wrong info